renderer/components/exports/export.tsx

```diff
@@ -0,0 +1,182 @@
+import electron from 'electron';
+import React, {useCallback, useMemo} from 'react';
+import classNames from 'classnames';
+
+import IconMenu from '../icon-menu';
+import {CancelIcon, MoreIcon} from '../../vectors';
+import {Progress, ProgressSpinner} from './progress';
+import useConversion from '../../hooks/editor/use-conversion';
+import {ExportStatus} from '../../common/types';
+import util from 'electron-util';
+import {useShowWindow} from '../../hooks/use-show-window';
+
+const stopPropagation = event => event.stopPropagation();
+
+const Export = ({id}: {id: string}) => {
+  const {state, isLoading, cancel, openInEditor, showInFolder, retry, copy} = useConversion(id);
+  useShowWindow(state?.id !== undefined);
+
+  const isCancelable = state?.status === ExportStatus.inProgress;
+  const isActionable = state?.filePath && !state?.disableOutputActions;
+  const canRetry = [ExportStatus.canceled, ExportStatus.failed].includes(state?.status);
+
+  const onCancel = () => {
+    cancel();
+  };
+
+  const onClick = () => {
+    showInFolder();
+  };
+
+  const fileNameClassName = classNames({
+    title: true,
+    disabled: !isActionable
+  });
+
+  const onDragStart = useCallback(event => {
+    event.preventDefault();
+    if (isActionable) {
+      electron.ipcRenderer.send('drag-export', state?.id);
+    }
+  }, [isActionable, state?.id]);
+
+  const menu = useMemo(() => {
+    return util.api.Menu.buildFromTemplate([{
+      label: 'Open Original',
+      click: openInEditor
+    },
+    state?.canCopy && {
+      label: 'Copy',
+      click: () => copy()
+    },
+    canRetry && {
+      label: 'Retry',
+      click: () => retry()
+    }].filter(Boolean));
+  }, [canRetry, retry, openInEditor]);
```

Comment:
Not sure about text/order/dividers for this. It's the context menu in the Exports window. At the moment it can have one of three states:

```
Open Original
```

```
Open Original
Copy
```

```
Open Original
Retry
```

Reply:
I think it should be:

```
Open Original
```

```
Copy
------
Open Original
```

```
Retry
------
Open Original
```